application/FtbListModel.cpp

```diff
@@ -88,7 +117,12 @@ QVariant FtbListModel::data(const QModelIndex &index, int role) const
 		}
 		return pack.description;
 	} else if(role == Qt::DecorationRole) {
-		//TODO: Add pack logos or something... but they have a weird size. This needs some design hacks
+		if(m_logoMap.contains(pack.logo)) {
+			return (m_logoMap.value(pack.logo));
+		}
+		QPixmap pixmap = MMC->getThemedIcon("screenshot-placeholder").pixmap(QSize(42, 42));
```

Comment:
`// FIXME: rewrite to use the QIcon directly, avoid resizing the images, set icon size in the view`